requests-for-comments/2023-10-25-per-product-nps.md

```diff
@@ -0,0 +1,47 @@
+Per-product NPS
+
+We would like to run permanent per-product NPS surveys in the PostHog app, for the following paid-for products:
+- Product Analytics
+- Feature Flags
+- A/B Testing
+- Session Replay
+- Surveys
+
+## Why?
+
+- Trend of customer satisfaction, we see how customer satisfaction develops over time. If we find that user satisfaction for a certain product is low or trending down, we can react
+- By asking follow-up questions to explain the score, we can learn which areas of the product in particular need improvement
+- By asking on a per-product basis, we gain more actionable feedback than having a blanket survey for general PostHog usage
+- The customer success team has an additional data point for individual customers if they are happy or not and can react quickly
+- NPS is an industry-known benchmark and allows us to compare ourselves to other products. The scale itself allows for more granularity than the [PMF survey](https://posthog.com/founders/measure-product-market-fit#indicator-2-pmf-survey), which is similarly known, but only offers a 3-point scale
+
+## Setup
+
+- 2-step survey, NPS and open-ended question for feedback
+- Link to draft survey for feature flags: https://app.posthog.com/surveys/018b665b-7396-0000-25ad-59cd7c843b4e
+- Question 1:
+    
+  <img width="346" alt="Screenshot 2023-10-25 at 14 28 21" src="https://github.com/PostHog/meta/assets/14750837/5c58d7f9-706f-4407-adcd-0d4571b3f460">
+    
+- Question 2:
+
+  <img width="343" alt="Screenshot 2023-10-25 at 14 29 05" src="https://github.com/PostHog/meta/assets/14750837/e2388e9b-0eb5-4749-ad18-00ef3ee2c3f2">
+  
+- Thank you: Get a free PostHog sticker (optional):      
+
+    <img width="350" alt="Screenshot 2023-10-25 at 14 29 55" src="https://github.com/PostHog/meta/assets/14750837/5661f99a-e0d3-470e-92ba-d0f3d83d53a8">
+
+  Note: There is a bug with the button, which is why there isn’t a more engaging CTA yet
+
+## Release conditions
+
+- Visitors on any feature flags page (app & eu). I am using the following regex: `https://(app|eu)\.posthog\.com/feature_flags`
```

Comment:
can we make it so that they only get it if they signed up 30 days ago or something? we really don't want to ask for nps score if someone just signed up...

Reply:
Good point, I think we can do this. I'll update the targeting

Reply:
@raquelmsmith Since I've updated the targeting to only target activated users, I have removed the 'signed up before' condition